src/treelearner/kernels/histogram_16_64_256.cu

```diff
@@ -0,0 +1,1059 @@
+/*!
+ * ibmGBT: IBM CUDA Accelerated LightGBM
+ *
+ * IBM Confidential
+ * Copyright (c) 2019 IBM Corporation. All rights reserved.
+ *
+ * The source code for this program is not published or otherwise
+ * divested of its trade secrets, irrespective of what has been
+ * deposited with the U.S. Copyright Office.
+ *
+ * US Government Users Restricted Rights - Use, duplication or
+ * disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
```

Comment:
I don't think that this copyright is acceptable in the OSS project with MIT licence..